app/workers/commit_monitor_handlers/commit_range/bugzilla_pr_checker.rb

```diff
@@ -32,7 +32,6 @@ def update_bugzilla_status(bug_id)
 
             add_pr_comment(bug)
             update_bug_status(bug)
-            bug.save
```

Comment:
The reason this was here was so that the PR comment and bug status go together as a single BZ POST (not sure that actually happened, but that was the intent.  I see this moves the save down to the update_bug_status.  Is there a similar save in add_pr_comment?

Reply:
Also answering https://github.com/ManageIQ/miq_bot/pull/434#discussion_r256490515 as well:

> Oh I see, so with the REST API it _can't_ be done in a single call?

So neither the REST API or the XMLRPC API was able to do this in a single call, but `active_bugzilla` has `active_model` included, so it just does a dirty state check on it's attributes, and if there is no attributes changed, it just no-ops.

https://github.com/ManageIQ/active_bugzilla/blob/3672da87bc25ce057978cf8fed9688670c09a50a/lib/active_bugzilla/bug.rb#L34-L39

Since we already do a check if we are going to do an update down in `update_bug_status`, I just moved the `bug.save` down to there since that is the only place we actually call a `.save`.